docs/sources/permissions/users.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Users"
+description = ""
+keywords = ["grafana", "configuration", "documentation", "admin", "users", "permissions"]
+type = "docs"
+[menu.docs]
+parent = "permissions"
+weight = 10
++++
+
+# Users
```

Comment:
This title does not really describe the content. I recommend putting the concept content back into the overview and then making each task a separate topic: add-a-user, remove-a-user, and so on. 

You would want at least a one-sentence intro in each, like "Add a new user account to your Grafana organization."